lib/control/v1/cubbyhole.js

```diff
@@ -7,11 +7,9 @@ function CubbyHole(storage) {
     const token = req.params.token;
     const path = req.params.path;
 
-    storage.lookup(token, path, CubbyHoleProvider, (err, result) => {
-      const response = (err) ? {error: err.message} : result;
-
-      res.json(response);
-    });
+    storage.lookup(token, path, CubbyHoleProvider)
+      .then((result) => res.json(result))
+      .catch((err) => res.json({error: err.message}));
```

Comment:
I think you may have meant to leave this comment on a `catch()` call in the `StorageService`, `LeaseManager`, or a `Provider` but the point is noted. 

We want a single error handler and have all errors defer their handling until `catch()`ed by it.